modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -99,6 +99,9 @@ def authorized_services
         if auth_services.include?(:directDepositBenefits) && direct_deposit_update_access?
           auth_services.push(:directDepositBenefitsUpdate)
         end
+        if Flipper.enabled?(:mobile_lighthouse_letters, @user) && auth_services.exclude?(:lettersAndDocuments)
+          auth_services.push(:lettersAndDocuments)
+        end
```

Comment:
Lighthouse has confirmed that we don't need to do any policy checking on Lighthouse. Only need to check there is an ICN. all our users will have ICN and we are doing a check for ICN just in case in the service object.